Automattic/wp-calypso
# Themes: Preview: Add "Try & Customize" button for modified sites

When #6349 was merged, it (intentionally) removed the "Try & Customize" button from both the Theme Sheet and the Theme Preview. Partially this was because if a site will be Headstarted (will have its content replaced by placeholder content) on theme switch, any changes made in the Customizer would actually prevent Headstart from running.

This PR restores the "Try & Customize" button as a *secondary* option in the preview, but only if the site would *not* be Headstarted. That is, sites with unmodified Headstart content will *not* see the button at all (although the option is still available from the ... menu in the showcase).

<img width="1017" alt="screen shot 2016-07-20 at 4 16 30 pm" src="https://cloud.githubusercontent.com/assets/2036909/17001525/578ee1b2-4e95-11e6-8568-7bcf2958aefb.png">

### Testing

- Visit the theme showcase, use the ... menu to preview a theme. Nothing should be different.
- Visit the ThemeSheet and click the "open live demo" link.
- If your site has been recently created and has no modified content, you should NOT see the "Try & Customize" button
- If your site has modified content, you should see two buttons, including "Try & Customize"
- Try the above for the active theme, non-active theme, single-site, multi-site, and logged-out to be sure everything works as expected. 
- The only views that should appear different are the single-site and multi-site view for a non-active theme.
